### PR TITLE
Add information about what we're doing

### PR DESCRIPTION
**Pull request by joshprzybyszewski**
Notice it's for HackISU, we're pretty simple, talk about the app design, and discuss targetted audience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # america-runs
 A project developed by four donut-loving lazy guys.
+We decided to build this for HackISU Spring 2017, a code jam from Friday night until Sunday morning. A simple app that utilizes Garmin's public SDK for monkeyC and their fitness watches
+
+## What it does
+Garmin has a bunch of fitness watches. Our app will run on *most* Garmin watches. The app will be a simple watch face displaying 3 things:
+1) The time
```
Comment (joshprzybyszewski) [inline README.md]: Oops. This should be `1.` not `1)`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+2) Some donuts
+3) A distance
+
+## What it means
+1) The time is the current time: simply a watch
```
Comment (joshprzybyszewski) [inline README.md]: ditto

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+2) The number of donuts on screen is equivalent to the number of calories you would burn running to and back from the nearest Dunkin Donuts
+3) The distance is the distance to the nearest Dunkin Donuts location
+
+## Target Audience
```
Comment (andrewvogler123) [inline README.md]: :+1:

**Review by joshprzybyszewski — COMMENTED** [review]
Fix the simple things you weirdo

**Comment by andrewvogler123** [comment]
+1

**Pull request merged**